pkgs/tools/misc/zellij/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, rustPlatform, cargo-make, stdenv, Security, SystemConfiguration, libiconv }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "zellij";
+  version = "0.5.0-beta";
+
+  src = fetchFromGitHub {
+    owner = "zellij-org";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0mhxq0pr87lz36wfn3g9yvgx5k6xwf69yvvjvrm3vf1l6hwm9nf6";
+  };
+
+  buildInputs = [ cargo-make ]
+    ++ lib.optionals stdenv.isDarwin [ Security SystemConfiguration libiconv ];
+
+  buildPhase = "cargo make";
```

Comment:
```suggestion
```

That will most likely not work and break the build in nix. @danieldk any idea how to build using cargo make?